api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -105,15 +105,15 @@ public void validate(Object target, Errors errors) {
 		// check attributes
```

Comment:
What does this have to do with TRUNK-6257 - Add a function to VisitService to get visits that overlap with a specified time period ?